src/components/Dashboard/TodoList.vue

```diff
@@ -561,13 +590,18 @@ export default {
       }
     },
 
-    loadAnnualReport (task) {
+    async loadAnnualReport (task) {
       let date
       const filing = task.task.filing
       if (filing && filing.header && filing.annualReport) {
         filing.annualReport.annualReportDate
           ? date = filing.annualReport.annualReportDate
           : date = filing.annualReport.nextARDate
+        const bcolErr = filing.header.paymentErrorType || null
+        let bcolObj = null
+        if (bcolErr) {
+          bcolObj = await this.getErrorObj(bcolErr)
+        }
```

Comment:
If you don't like the let and the separate if statement, in Javascript you can do cool things like this:
`const bcolObj = bcolErr && await this.getErrorObj(bcolErr)`

Reply:
I like it!

Reply:
Fixed.